docs/source/conf.py

```diff
@@ -162,6 +164,9 @@
     ]
 }
 
+html_baseurl = 'https://docs.conda.io/projects/conda/'
+sitemap_locales = [None]
+sitemap_url_scheme = "{lang}stable/{link}"
```

Comment:
We're hard-coding stable here since that's what we want Google to point to.

Reply:
should probably add this as an inline comment?

Reply:
True!